OTAnalytics/application/use_cases/reset_state.py

```diff
@@ -0,0 +1,9 @@
+from OTAnalytics.application.state import VideosMetadata
+
+
+class ResetState:
+    def __init__(self, videos_metadata: VideosMetadata) -> None:
+        self._videos_metadata = videos_metadata
+
+    def reset(self) -> None:
```

Comment:
Do we also need to reset , TrackViewState, TrackState, SectionState, FlowState, TracksMetadata, ActionState, FileState?